src/ceph-osd-prestart.sh

```diff
@@ -21,15 +21,14 @@ data="/var/lib/ceph/osd/${cluster:-ceph}-$id"
 journal="$data/journal"
 
 if [ -L "$journal" -a ! -e "$journal" ]; then
+    echo "ceph-osd(${cluster:-ceph}-$id): journal not present, not starting yet." 1>&2
     udevadm settle --timeout=5 || :
     if [ -L "$journal" -a ! -e "$journal" ]; then
-        echo "ceph-osd($UPSTART_INSTANCE): journal not present, not starting yet." 1>&2
-        stop
-        exit 0
+        echo "ceph-osd(${cluster:-ceph}-$id): journal still not present, aborting." 1>&2
+        exit 1
```

Comment:
@b-ranto So, change this line to "exit 0" and perhaps change the warning in the previous line to "journal device still not present after 5 seconds; giving up" ?